Gemfile

```diff
@@ -2,34 +2,27 @@ source 'https://rubygems.org'
 
 ruby "3.3.8"
 
-gem "rails", "~> 6.1.7"
-
-# TODO Consider removing these after upgrading Rails. These are a Ruby 3.3 => 3.4 warning.
-# add required gems leaving standard library to avoid log noise
-gem "bigdecimal"
-gem "drb"
-gem "mutex_m"
-gem "observer"
-
+gem "rails", "8.1.0.beta1"
 
 gem 'sprockets-rails', '~> 3.2.2'
-gem "activerecord-trilogy-adapter" # for mysql installation
+gem 'trilogy'
```

Comment:
wasn't this already in there?

Reply:
I think trilogy was indirectly loaded as a dependency for activerecord-trilogy-adapter. That is no longer needed as of rails 7.1+, so I added just the trilogy adapter.